src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -449,7 +446,8 @@ function (SpanData $span) use ($controllerName) {
                         return;
                     }
                     if (!$injectedActionInfo) {
-                        if ($integration->injectActionInfo($event, $eventName, $integration->symfonyRequestSpan)) {
+                        $rootSpan = root_span();
+                        if ($rootSpan !== null && $integration->injectActionInfo($event, $eventName, $rootSpan)) {
```

Comment:
```suggestion
                        if ($integration->injectActionInfo($event, $eventName, root_span())) {
```
Redundant: in trace_method